src/js/components/List/List.js

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import { compose } from 'recompose';
+import styled, { css, withTheme } from 'styled-components';
+
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { InfiniteScroll } from '../InfiniteScroll';
+import { Keyboard } from '../Keyboard';
+import { Text } from '../Text';
+import { backgroundStyle, focusStyle, normalizeColor } from '../../utils';
+import { withFocus, withForwardRef } from '../hocs';
+
+const StyledList = styled(Box)`
```

Comment:
why are we using Box instead of the `ol` directly?

Reply:
good idea. Changed to `ul`